snapcraft/internal/meta.py

```diff
@@ -221,6 +252,25 @@ def _wrap_apps(self, apps):
                             str(e), app))
         return apps
 
+    def _wrap_hooks(self, hooks):
+        for hook in hooks:
+            try:
+                command = hooks[hook]['command']
+
+                # The 'command' is for snapcraft only, not snapd. Remove it,
+                # and if that makes the dict empty, set it to None instead.
+                del hooks[hook]['command']
+                if len(hooks[hook]) == 0:
+                    hooks[hook] = None
```

Comment:
I have a dict of dicts. I'm removing an item from a sub-dict, not the dict being traversed. Is that still not okay?